Assets/ConsentManagementProvider/Plugins/iOS/Source/SwiftBridge.swift

```diff
@@ -164,14 +164,23 @@ import UIKit
     }
 
 // MARK: - Manage lib
-    @objc public func loadMessage(authId: String? = nil) {
+    @objc public func loadMessage() {
```

Comment:
@Nevazhnovu I think i't safe to keep a single `loadMessage` signature like:
```swift
@objc public func loadMessage(authId: String? = nil)
```

Because in the file `Assets/ConsentManagementProvider/Scripts/wrapper/iOS/ConsentWrapperIOS.cs` you're checking if `authId` is null and calling the appropriate objc method:
```cs
            if (string.IsNullOrEmpty(authId))
            {
                CmpDebugUtil.Log("Calling load message without authId");
                _loadMessage();
            }
            else 
            {
                CmpDebugUtil.Log($"Calling load message with authId={authId}");
                _loadMessageWithAuthId(authId);
            }
 ```

Reply:
@andresilveirah Done!